### ISSUE TITLE
AttributeError when cleaning simulated data

### ISSUE THREAD

**Issue by samuelskoda** [tlamadon/bipartitepandas#15]
Hi,

I am just starting to use the package with simulated data using the example code, however I cannot get beyond cleaning - when I simulate data using the example and cleaning parameters, I get the following error:

```
>>>bdf=bdf.clean(clean_params)
checking required columns and datatypes
sorting rows
dropping NaN observations
generating 'm' column
keeping highest paying job for i-t (worker-year) duplicates (how='max')
dropping workers who leave a firm then return to it (how='returners')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/ssb/bruker/sml/.local/lib/python3.6/site-packages/bipartitepandas/bipartitelong.py", line 100, in clean
    frame = super().clean(params)
  File "/ssb/bruker/sml/.local/lib/python3.6/site-packages/bipartitepandas/bipartitelongbase.py", line 154, in clean
    frame = frame._drop_returns(how=drop_returns, is_sorted=True, reset_index=True, copy=False)
  File "/usr/local/lib64/python3.6/site-packages/pandas/core/generic.py", line 5179, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute '_drop_returns'
(4 lines skipped)
```

When I instead load fake dataset from leed2way, the error occurs one step earlier with the AttributeError: 'DataFrame' object has no attribute 'gen_m'.

**Comment by adamoppenheimer** [comment]
Hi Samuel,

Thanks for using the package!

I'm sorry about the issue you are having, it looks similar to [this issue](https://github.com/tlamadon/pytwoway/issues/50#issuecomment-1734382820).

Could you please let me know if downgrading to Pandas 2.0.3 fixes your issue?

Thank you!

Best,
Adam

**Comment by adamoppenheimer** [comment]
I just uploaded BipartitePandas v1.1.10 which should fix this bug by requiring pandas <= 2.0.3, please mark this as resolved if your code works after updating.

If your code still isn't working, let me know so we can try to figure out how to get it to work.

Thanks!

Best,
Adam

**Comment by samuelskoda** [comment]
Thank you for the quick response and the amazing package! The issue was that the Linux server was running Python 3.6 that allows only pandas 1.1.5. I installed locally recent miniforge3 version and now with pandas 2.0.3 the code runs.
